lib/replicator.js

```diff
@@ -386,6 +386,21 @@ class Peer {
     if (drop) this._unclearLocalRange(start, length)
     else this._clearLocalRange(start, length)
 
+    // TODO: drop===true case
```

Comment:
I think it's fine to ignore the `drop===true` case for now, and have it behave as before. We can then iterate on that later if we notice it becomes relevant